ck.sh

```diff
@@ -77,6 +77,10 @@ case "$COMMAND" in
 				# Replace with new build files.
 				cp -r dev/builder/release/ckeditor/* ../ckeditor/
 
+				(cat ../node_modules/jquery/dist/jquery.min.js && cat ../ckeditor/plugins/a11ychecker/plugin.js) > ../ckeditor/plugins/a11ychecker/plugin.jquery.js
```

Comment:
`cat` takes multiple arguments ("cat" comes from "concatenate").

So, `cat x && cat y` is better written as `cat x y`.

Reply:
Thanks @wincent,
This is just a temporary hack and I didn't really think about writing "idiomatic" shell script but I appreciate the suggestion.